adminlogin.aspx.cs

```diff
@@ -9,9 +11,45 @@ namespace LogBook
 {
     public partial class adminlogin : System.Web.UI.Page
     {
+        string strcon = ConfigurationManager.ConnectionStrings["con"].ConnectionString;
+
         protected void Page_Load(object sender, EventArgs e)
         {
 
         }
+
+        protected void Button1_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                SqlConnection con = new SqlConnection(strcon);
+                if (con.State == System.Data.ConnectionState.Closed)
+                {
+                    con.Open();
+                }
+                SqlCommand cmd = new SqlCommand("select * from admin_login_tbl where username='" + TextBox1.Text.Trim() + "' AND password='" + TextBox2.Text.Trim() + "'", con);
```

Comment:
### Poor coding standards

**SQL keywords should be capitalized**
```suggestion
                SqlCommand cmd = new SqlCommand("SELECT * FROM admin_login_tbl WHERE username='" + TextBox1.Text.Trim() + "' AND password='" + TextBox2.Text.Trim() + "'", con);
```
**Make your code more readable**
Instead of having everything in one line, split it up into multiple lines for readability's sake. 

```suggestion
                string selectQuery = $@"SELECT * FROM admin_login_tbl";`
                SQLCommand cmd = new SQLCommand(selectQuery, con);
```